packages/synapse-react-client/src/components/PortalSectionHeader/PortalSectionHeader.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Stack,
+  Typography,
+  Button,
+  SxProps,
+  Theme,
+  Box,
+  Link as MuiLink,
+} from '@mui/material'
+import { Link as RouterLink } from 'react-router'
+import { spreadSx } from '../../theme/spreadSx'
+
+export type PortalSectionHeaderProps = {
+  title: string
+  buttonText?: string
+  summaryText?: React.ReactNode
+  link?: string
+  sx?: SxProps<Theme>
+  centered?: boolean
+  reverseButtonAndText?: boolean
+}
+
+const PortalSectionHeader = ({
+  title,
+  buttonText,
+  summaryText,
+  link,
+  sx,
+  centered = false,
+  reverseButtonAndText = false,
+}: PortalSectionHeaderProps) => {
+  const iisExternalLink =
+    link?.startsWith('http://') || link?.startsWith('https://')
+  return (
+    <Box
+      sx={spreadSx(sx, {
+        ...(centered && {
+          width: '100%',
+          display: 'flex',
+          justifyContent: 'center',
+        }),
+      })}
+    >
+      <Stack
+        sx={{
+          gap: '16px',
+          borderTop: '4px solid',
+          borderColor: 'grey.400',
+          alignItems: centered ? 'center' : 'flex-start',
+        }}
+      >
+        <Typography
+          variant="headline2"
+          paddingTop="30px"
+          color="grey.1000"
+          fontSize={{ xs: '24px', md: '32px' }}
+        >
+          {title}
+        </Typography>
```

Comment:
As mentioned elsewhere, I suggest moving the border to the h2, and in components, make h2 100% width when the border should take up the full width of the section
```suggestion
      <Stack
        sx={{
          gap: '16px',
          alignItems: centered ? 'center' : 'flex-start',
        }}
      >
        <Typography
          variant="headline2"
          paddingTop="30px"
          color="grey.1000"
          fontSize={{ xs: '24px', md: '32px' }}
          sx={{
            borderTop: '4px solid',
            borderColor: 'grey.400',
          }}
        >
          {title}
        </Typography>
```